staff/marti-malek/node-unix-commands/cp.js

```diff
@@ -0,0 +1,8 @@
+const fs = require('fs')
+
+const { argv: [, , from, to] } = process
+
+fs.copyFile(from, to, err => {
```

Comment:
You could try to do it without the copyFile method! And also you could try to do it recursive or not! 🤓